### ISSUE TITLE
How about the final performance of the implemented bbfcn?

### ISSUE THREAD

**Issue by mariolew** [nrailgun/bbfcn#1]
Have you tested the performance of bbfcn you implemented?

**Comment by nrailgun** [comment]
No, I was training it with an NV Titan, and I thought it's acceptable. Actually I found the parameter tuning is so hard.
